### DISCUSSION TITLE
Tampilan di shiny saya yang dashboardpage nya "category" koq nggk came out ya,,,

### DISCUSSION THREAD

**Discussion by SitiNurbaya28** [teamalgoritma/community#717] [Q&A] [answered]
dashboardpage saya  yang untuk "category" nggk ada outputnya,,
berikut tampilan d dashboard, console, dan UI.R,,server.R nya

![Screenshot 2023-07-05 205207](https://github.com/teamalgoritma/community/assets/133105934/aa2eb13a-3486-4944-b9a9-86b99a01596a)
![Screenshot 2023-07-05 204233](https://github.com/teamalgoritma/community/assets/133105934/5da33a67-03b8-4425-876e-5eee087f4448)
![Screenshot 2023-07-05 204438](https://github.com/teamalgoritma/community/assets/133105934/b77504ad-98df-4dac-8cd3-233b697ec9e9)
![Screenshot 2023-07-05 204543](https://github.com/teamalgoritma/community/assets/133105934/0e875098-2d88-439c-b272-55d727ec71aa)

**Comment by IrfanChairurrachman** [comment]
Dear @SitiNurbaya28 ,

Terima kasih atas pertanyaannya, akan kami coba telusuri ya Bu permasalahannya

Terima kasih

**Reply by divakartika** [reply]
Dear @SitiNurbaya28 ,

Boleh dilampirkan ketika halaman Category diklik, tampilan seperti apa yang muncul?

Terima kasih

**Reply by SitiNurbaya28** [reply]
dear mba diva,

itu dia pas d klik dia nggk muncul sama sekali,,

**Reply by SitiNurbaya28** [reply]
notif di console nya seperti yang di lampirkan di atas mba diva,,,

**Reply by SitiNurbaya28** [reply, marked as answer]
mba diva,mas irfan,,dashboard nya udah bisa keluar ouput ,,ternyata ada yang salah di penulisan fungsi di RMD nya,,,

maaf udah ngereotin mba diva,,mas irfan,,
makasih banyak udah mau bales chat saya,,,

**Reply by IrfanChairurrachman** [reply]
Dear @SitiNurbaya28 ,

Baik Bu, terima kasih atas konfirmasinya, kami izin tutup diskusi ini ya Bu.

Terima kasih.
